Makefile

```diff
@@ -36,6 +36,8 @@ UTC_DATE=$(shell date -u "+%Y-%m-%d")
 
 GO_VERSION := $(shell cat GO_VERSION)
 
+ARCH=$(subst aarch64,arm64,$(subst x86_64,amd64,$(patsubst i%86,386,$(shell uname -m))))
```

Comment:
Maybe just use `$(GO) env GOARCH`  here to avoid the translation of architecture names and avoid having to adjust it for new architectures?

Reply:
Good comment, thanks